### PR TITLE
statistics: add more write query kind

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

add more write query kind for hot region scheduler

### What is changed and how it works?

insert 1200 tables
image

write test with Pessimistic

image

image

### Check List

Tests

- Unit test

Related changes

- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by nolouch — APPROVED** [review]

File: server/core/region_option.go [hunk 1]
```diff
@@ -201,9 +201,13 @@ func SetReadQuery(v uint64) RegionCreateOption {
...
-		Put:         v / 3,
-		Delete:      v / 3,
-		DeleteRange: v / 3,
+		Put:                    v / 7,
+		Delete:                 v / 7,
+		DeleteRange:            v / 7,
+		Rollback:               v / 7,
+		Prewrite:               v / 7,
+		Commit:                 v / 7,
+		AcquirePessimisticLock: v / 7,
```
Comment (bufferflies) [inline server/core/region_option.go:210]: why use seven field to record the same query stat ??

File: server/statistics/kind_test.go [hunk 2]
```diff
@@ -27,12 +27,16 @@ type testRegionInfoSuite struct{}
 
 func (s *testRegionInfoSuite) TestGetLoads(c *C) {
 	queryStats := &pdpb.QueryStats{
-		Get:         5,
-		Coprocessor: 6,
-		Scan:        7,
-		Put:         8,
-		Delete:      9,
-		DeleteRange: 10,
+		Get:                    5,
```
Comment (bufferflies) [inline server/statistics/kind_test.go:30]: the test is not same with the state set??

File: server/core/region_option.go [hunk 3]
```diff
@@ -201,9 +201,13 @@ func SetReadQuery(v uint64) RegionCreateOption {
 // SetWrittenQuery sets the write query for the region.
 func SetWrittenQuery(v uint64) RegionCreateOption {
```
Comment (Yisaer) [inline server/core/region_option.go:202]: Is this method should only be used in test?

File: server/statistics/kind_test.go (hunk 2 above)
Comment (lhy1024) [reply server/statistics/kind_test.go:30]: only set different num

File: server/core/region_option.go (hunk 1 above)
Comment (lhy1024) [reply server/core/region_option.go:210]: I will fill one of them randomly later

File: server/statistics/kind_test.go [hunk 4]
```diff
@@ -27,12 +27,16 @@ type testRegionInfoSuite struct{}
...
+		Coprocessor:            6,
+		Scan:                   7,
+		Put:                    8,
+		Delete:                 9,
+		DeleteRange:            10,
+		AcquirePessimisticLock: 11,
+		Rollback:               12,
+		Prewrite:               13,
+		Commit:                 14,
```
Comment (rleungx) [inline server/statistics/kind_test.go:39]: Do we need to consider `PrewritePessimistic`?

Comment (lhy1024) [reply server/statistics/kind_test.go:39]: yes, https://github.com/tikv/tikv/blob/1e4a234ffc682804686aec896e4bdbab0b1dfb63/src/storage/txn/scheduler.rs#L653

File: server/core/region_option.go (hunk 3 above)
Comment (lhy1024) [reply server/core/region_option.go:198]: yes

Comment (rleungx) [reply server/core/region_option.go:198]: Please add a comment.

**Review by Yisaer — APPROVED** [review]

**Pull request merged**